packages/sw-precaching/src/lib/controllers/base-cache-manager.js

```diff
@@ -118,10 +119,15 @@ class BaseCacheManager {
       return;
     }
 
-    let response = await fetch(precacheEntry.getNetworkRequest(), {
+    let response = await this._requestWrapper.fetch({
```

Comment:
There is one problem with that. The current precaching code will fetch request with the cache busting and then cache it against the origin URL to make future cache.matches possible with the original URL. fetchAndCache will just cache the URL with the cache busting.